client/auth/emailed-login-link-expired.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from 'components/empty-content';
+import RedirectWhenLoggedIn from 'components/redirect-when-logged-in';
+
+class EmailedLoginLinkExpired extends React.Component {
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<RedirectWhenLoggedIn
+					redirectTo="/"
+					replaceCurrentLocation={ true }
+				/>
+				<EmptyContent
+					action={ translate( 'Return to WordPress.com' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **16** times:
`translate( 'Return to WordPress Updates page' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).